src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -118,9 +118,7 @@ static PackedFlags()
 
                 // 1) Verify that the range of method kinds doesn't fall outside the bounds of the
                 // method kind mask.
-                var methodKinds = EnumUtilities.GetValues<MethodKind>();
-                var maxMethodKind = (int)System.Linq.Enumerable.Aggregate(methodKinds, (m1, m2) => m1 | m2);
-                Debug.Assert((maxMethodKind & MethodKindMask) == maxMethodKind);
+                Debug.Assert(EnumUtilities.ContainsAllValues<MethodKind>(MethodKindMask));
```

Comment:
>Debug.Assert(EnumUtilities.ContainsAllValues<MethodKind>(MethodKindMask)); [](start = 16, length = 74)

The assert doesn't look equivalent. In particular, it will allow the ```MethodKindMask``` to have more bits set than in all ```MethodKind``` values or-ed .

Reply:
That was possible previously.

---
In reply to: [286254251](https://github.com/dotnet/roslyn/pull/35846#discussion_r286254251) [](ancestors = 286254251)